ARC specification.md

```diff
@@ -296,102 +302,156 @@ Notes:
-- the target MUST create a `badge.svg` file that visually summarizes the results of validating the ARC against the cases defined in the target. The information displayed SHOULD be derivable from the `validation_report.*` file and MUST include the _name_ of the target.
+## Validation
 
-### Structure of the validation branch
+The process of assessing quality parameters of an ARC is further referred to as _validation_ of the ARC against a [_validation package_](#validation-packages), where the _validation package_ is an arbitrary set of [validation cases](#validation-cases) that the ARC MUST pass to qualify as _valid_ in regard to the _validation package_.
 
-To make sure that validation results are bundled with the ARC but do not pollute the commit history, validation results MUST be stored in a separate branch of the ARC repository. This branch:
-- MUST be named `validation`
+### Validation cases
+
+A **validation case** is the atomic unit of a [validation package](#validation-packages) describing a single, deterministic and reproducible requirement that the ARC MUST satisfy in order to qualify as _valid_ in regard to it.
+
+Format and scope of these cases naturally vary depending on the type of ARC, aim of the containing validation package and tools used for creating and performing the validation. 
+Therefore, no further requirements are made on the format of validation cases.
+
+  example:
+
+  > The following example shows a validation case simply defined using natural language.
+
+  ```
+  All Sample names in this ARC must be prefixed with the string "Sample_"
+  ```
+
+  Any ARC where all sample names are prefixed with the string "Sample_" would be considered valid in regard to this validation case.
+
+### Validation packages
+
+A **validation package** bundles a collection of [validation cases](#validation-cases) that the ARC MUST pass to qualify as _valid_ in regard to the _validation package_ with instructions on how to perform the validation and summarize the results.
+
+Validation packages
+
+- MUST be executable. 
+  This can for example be achieved by implementing them in a programming language, a shell script, or a workflow language.
+
+- MUST validate an ARC against all contained validation cases upon execution.
+
+- MUST have a globally unique name.
+  This will eventually be enforced by a central validation package registry
+
+- SHOULD be versioned using [semantic versioning](https://semver.org/)
+
+- MUST create a `validation_report.*` file upon execution that summarizes the results of validating the ARC against the cases defined in the validation package.
+  The format of this file SHOULD be of an established test result format such as [JUnit XML](https://github.com/windyroad/JUnit-Schema) or [TAP](https://testanything.org/).
+
+- MUST create a `badge.svg` file upon execution that visually summarizes the results of validating the ARC against the validation cases defined in the validation package.
```

Comment:
Any requirements of the shape of the contained visual summary, (e.g. according to badge.io formats)?

Reply:
You mean shields.io? no i think as long as this is a svg everything is fine

Reply:
Ah yes 😅 
Alright